tests/acceptance/reminder-list-test.js

```diff
@@ -29,3 +29,14 @@ test('clicking on an individual item', function(assert) {
     assert.equal(Ember.$('.reminder-item:first').text().trim(), Ember.$('.reminder-title').text().trim());
   });
 });
+
+test('Display prompt if no reminders are present', function(assert) {
+
+  visit('/reminders');
+
+  andThen(function() {
+    assert.equal(Ember.$('.no-reminders-notice').length, 1);
```

Comment:
I'm copy-pasta coding. I've found it's better to trust that Ember doing something magical than try to forge something myself. This puts me into a 'if it works, don't change it' mindset.